app/hotels/src/singleHotel/bookNow/BookNow.js

```diff
@@ -40,9 +42,37 @@ export class BookNow extends React.Component<Props> {
     }
   };
 
+  onPress = () => {
+    const numberOfGuests = this.props.getGuestCount();
+    if (numberOfGuests > this.props.maxPersons) {
+      Alert.translatedAlert(
+        {
+          id: 'single_hotel.book_now.alert_title',
+        },
+        {
+          id: 'single_hotel.book_now.alert_body',
+          values: { numberOfGuests },
+        },
+        [
+          {
+            text: { id: 'shared.button.cancel' },
+          },
```

Comment:
```suggestion
            style: 'destructive',
          },
```

Reply:
It would make UX better in my opinion.
![simulator screen shot - iphone x - 2019-01-07 at 16 06 21](https://user-images.githubusercontent.com/22741774/50775341-311eb080-1296-11e9-96ea-dd3c90353959.png)